docs/workflow/testing/libraries/testing.md

```diff
@@ -1,47 +1,62 @@
 # Testing Libraries
 
-We use the OSS testing framework [xunit](https://github.com/xunit/xunit).
+## Pre-requisites
+Before tests can run the following steps must happen:
+1. Build a runtime (either Mono or coreclr)
+2. Build the libraries
+3. Testhost deployment: Binaries must be arranged into a directory layout that tools/tests expect
+4. Build the tests
 
-To build the tests and run them you can call the libraries build script. For libraries tests to work, you must have built the coreclr or mono runtime for them to run on.
+Using build.cmd/sh these steps can either be accomplished independently or the testing and all the pre-requisites can be automated in a single command. See the [Building instructions](../../building/libraries/README.md) for more options on how to build pieces independently.
 
-**Examples**
-- The following shows how to build only the tests but not run them:
+## All-in-one Examples
+
+These examples automate all pre-requisite steps and the test run in a single command from a clean enlistment.
+
+- Run all tests using clr:
 ```
-build.cmd/sh -subset libs.tests
+build.cmd/sh -subset clr+libs -test
```

Comment:
```suggestion
build.cmd/sh -subset clr+libs+libs.tests -test
```

Reply:
Can you please double check that command actually works? I believe it's missing the `-restore` and `-build` arguments. You should be able to test that from a clean build.

Reply:
In the old docs it said "Unless you specifiy `-testnobuild`, test assemblies are implicitly built when invoking the `Test` action" which I took to mean that "libs.tests" was not required here. Was that inaccurate or I misunderstood?

Reply:
That is true but both `clr` and `libs` don't react to the `-test` action.

Reply:
Ah, that definitely mislead me but I think I understand now. I'll update accordingly.